tests/integ/scala/test_view_suite.py

```diff
@@ -215,3 +217,38 @@ def test_create_temp_view_on_functions(session, local_testing_mode):
         Utils.drop_table(session, table_name)
         if not local_testing_mode:
             Utils.drop_view(session, view_name)
+
+
+@pytest.mark.skipif(IS_IN_STORED_PROC_LOCALFS, reason="need resources")
+def test_create_or_replace_temp_view_with_file(session, resources_path):
```

Comment:
can we add a `@multithreaded_run` here to ascertain this is thread-safe

Reply:
done